src/api/telegram-subscription/routes/telegram-subscription.ts

```diff
@@ -22,13 +22,22 @@ const customRouter = (innerRouter, extraRoutes = []) => {
 const myExtraRoutes = [
   {
     method: 'GET',
-    path: '/tg-subscriptions/:account',
+    path: '/tg-subscriptions',
```

Comment:
I think we should revisiting the naming of the endpoints to be more rest API like

Reply:
Changed to `/subscriptions/telegram`